src/main/java/EtfRecommendService/webSocket/WebSocketConfig.java

```diff
@@ -0,0 +1,38 @@
+package EtfRecommendService.webSocket;
+
+import org.springframework.context.annotation.Configuration;
+import org.springframework.messaging.simp.config.MessageBrokerRegistry;
+import org.springframework.web.socket.config.annotation.*;
+
+
+public class WebSocketConfig implements WebSocketConfigurer {
```

Comment:
`@Configuration`이 누락되지 않았나 싶네요
```suggestion
@Configuration
public class WebSocketConfig implements WebSocketConfigurer {
```